packages/client/index.js

```diff
@@ -67,6 +67,7 @@ async function buildOpenAPIClient (options, openTelemetry) {
     baseUrl = computeURLWithoutPath(options.url)
   }
 
+  client.operationIdMap = {}
```

Comment:
This will unfortunately conflict with an operation called `operationIdMap`...
I know this is a far off concern, but it might be problematic long term.
The solution I proposed was to have it as an additional property of the functions,
so one could do:

```js
Object.keys(client).reduce((acc, name) => {
  const { method, path } = client[name]
  acc[name] = { method, path }
  return acc
}, {})
```

Reply:
Okay, I understand the concern. What if I use a `Symbol`?

Reply:
Also, `client` is already containing a `Symbol(headers)]` 😉
So, I like the idea of adding a new one, since it will avoid future conflicts in case...

Reply:
A Symbol is ok

Reply:
Cool, I'll do the change 👍🏼

Reply:
Done on [d82fdde](https://github.com/platformatic/platformatic/pull/1837/commits/d82fdde330b74bf856dd4d3f5ad3a864c39c3814)